tests/integration/mci/test_happy_path.py

```diff
@@ -24,6 +24,7 @@ def happy_path(self,form_type_id):
         content = resp.get_data(True)
         self.assertRegexpMatches(content, '<title>Introduction</title>')
         self.assertRegexpMatches(content, '>Get Started<')
+        self.assertRegexpMatches(content, '<div class="bar__title">Monthly Business Survey - Retail Sales Index</div>')
```

Comment:
Is it a good idea to include markup in our tests? If I change the markup it'll break the test in several places.